src/core/time.d

```diff
@@ -292,7 +292,7 @@ else version(Darwin) enum ClockType
     precise = 3,
     second = 6,
 }
-else version(linux) enum ClockType
+else version(CRuntime_Glibc) enum ClockType
```

Comment:
This is going to break this module for Bionic and the tests below for Musl.  I see no reason to change this, even if these constants aren't defined for you.